provider/schema/resolvers_test.go

```diff
@@ -129,29 +129,35 @@ var networkTestTable = &Table{
 			Name: "net",
 			Type: TypeCIDR,
 		},
+		{
+			Name: "ips",
+			Type: TypeInetArray,
+		},
 	},
 }
 
 type testNetStruct struct {
 	IP  string
 	MAC string
 	Net string
+	IPS []string
 }
 
 var netTests = []testNetStruct{
-	{IP: "192.168.1.12", MAC: "2C:54:91:88:C9:E3", Net: "192.168.0.1/24"},
-	{IP: "2001:0db8:85a3:0000:0000:8a2e:0370:7334", MAC: "2C-54-91-88-C9-E3", Net: "2002::1234:abcd:ffff:c0a8:101/64"},
-	{IP: "::1234:5678", MAC: "2C-54-91-88-C9-E3", Net: "::1234:5678/12"},
+	{IP: "192.168.1.12", MAC: "2C:54:91:88:C9:E3", Net: "192.168.0.1/24", IPS: []string{"192.168.1.12"}},
+	{IP: "2001:0db8:85a3:0000:0000:8a2e:0370:7334", MAC: "2C-54-91-88-C9-E3", Net: "2002::1234:abcd:ffff:c0a8:101/64", IPS: []string{"2001:0db8:85a3:0000:0000:8a2e:0370:7334"}},
+	{IP: "::1234:5678", MAC: "2C-54-91-88-C9-E3", Net: "::1234:5678/12", IPS: []string{"::1234:5678"}},
 }
 var netTestsFails = []testNetStruct{
-	{IP: "192.168.1/", MAC: "2C:54:91:88:C9", Net: "192.168.0.1-24"},
-	{IP: "::1234:5678:", MAC: "2C:54-91-88-C9-E3", Net: "2002::1234:abcd:ffff:c0a8:101-64"},
+	{IP: "192.168.1/", MAC: "2C:54:91:88:C9", Net: "192.168.0.1-24", IPS: []string{"192.168.1/"}},
+	{IP: "::1234:5678:", MAC: "2C:54-91-88-C9-E3", Net: "2002::1234:abcd:ffff:c0a8:101-64", IPS: []string{"::1234:5678:"}},
```

Comment:
maybe add more than 1 IP to the in the array. In the failing tests add 1 good IP one Bad IP